HudiConnector/src/main/java/basic/application/StreamingJob.java

```diff
@@ -0,0 +1,141 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * This file has been extended from the Apache Flink project skeleton.
+ */
+
+package basic.application;
+import com.amazonaws.services.kinesisanalytics.runtime.KinesisAnalyticsRuntime;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.api.EnvironmentSettings;
+import org.apache.flink.api.java.utils.ParameterTool;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.Properties;
+import java.util.Map;
+
+/**
+ * Skeleton for a Hudi Flink Streaming Job.
+ *
+ * <p>For a tutorial how to write a Flink streaming application, check the
+ * tutorials and examples on the <a href="https://flink.apache.org/docs/stable/">Flink Website</a>.
+ *
+ * <p>To package your application into a JAR file for execution, run
+ * 'mvn clean package' on the command line.
+ *
+ * <p>If you change the name of the main class (with the public static void main(String[] args))
+ * method, change the respective entry in the POM.xml file (simply search for 'mainClass').
+ *
+ * <p>Disclaimer: This core is not production ready.</p>
+ */
+public class StreamingJob {
+	private static final Logger LOG = LoggerFactory.getLogger(StreamingJob.class);
+
+	public static void main(String[] args) throws Exception {
+		// set up the streaming execution environment
+
+		final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+		ParameterTool parameter;
+		parameter = ParameterTool.fromArgs(args);
+
+		//read the parameters from the Kinesis Analytics environment
+		Map<String, Properties> applicationProperties = KinesisAnalyticsRuntime.getApplicationProperties();
+		Properties flinkProperties = null;
+
+		String kafkaTopic = parameter.get("kafka-topic", "AWSKafkaTutorialTopic");
+		String brokers = parameter.get("brokers", "");
+		String s3Path = parameter.get("s3Path", "");
+
+		if (applicationProperties != null) {
+			flinkProperties = applicationProperties.get("FlinkApplicationProperties");
+		}
+
+		if (flinkProperties != null) {
+			kafkaTopic = flinkProperties.get("kafka-topic").toString();
+			brokers = flinkProperties.get("brokers").toString();
+			s3Path = flinkProperties.get("s3Path").toString();
+		}
+
+		LOG.info("kafkaTopic is :" + kafkaTopic);
+		LOG.info("brokers is :" + brokers);
+		LOG.info("s3Path is :" + s3Path);
+
+		//Create Properties object for the Kafka consumer
+		Properties kafkaProps = new Properties();
+		kafkaProps.setProperty("bootstrap.servers", brokers);
+
+		//Process stream using sql API
+		StreamingSQLAPI.process(env, kafkaTopic, s3Path , kafkaProps);
+	}
+
+
+	public static class StreamingSQLAPI {
+
+		public static void process(StreamExecutionEnvironment env, String kafkaTopic, String s3Path, Properties kafkaProperties)  {
```

Comment:
nit: The naming of the class and method are not intuitive here. If you must create a new class then `StreamingSQLAPI.process(..)` could be changed to something like `KafkaHudiSqlExample.createAndDeployJob(..)`